### PR TITLE
Joomla 3.9.10 merge

### PR DESCRIPTION
**Pull request by wilsonge**
- Merge with a merge not a squash/rebase

### CODE REVIEW

**Comment by wilsonge** [comment]
Known failure case trying to login into the admin site. So choosing to merge this with the failing test. Tests passed in mysql suggesting that the system tests are largely working and the only postgres changes here relate to template styles

EDIT: PostgreSQL passed on the merge commit drone run - so this was indeed just a blip

File: plugins/fields/repeatable/params/repeatable.xml [hunk 1]
```diff
@@ -30,6 +30,38 @@
 								<option value="text">PLG_FIELDS_REPEATABLE_PARAMS_FIELDNAME_TYPE_TEXT</option>
 								<option value="textarea">PLG_FIELDS_REPEATABLE_PARAMS_FIELDNAME_TYPE_TEXTAREA</option>
 							</field>
+							<field
+								name="fieldfilter"
+								type="list"
+								label="PLG_FIELDS_TEXT_PARAMS_FILTER_LABEL"
+								description="PLG_FIELDS_TEXT_PARAMS_FILTER_DESC"
```
Comment (brianteeman) [inline plugins/fields/repeatable/params/repeatable.xml:37]: this language string is not present in j4 because we removed the usage of the description

**Comment by brianteeman** [comment]
you merged as I was typing an issue. do you want me to submit a pr or will you add to the commit

**Comment by wilsonge** [comment]
Sorry - can you do a PR please. Just saw this and about to head to bed.

**Comment by brianteeman** [comment]
done #26112

**Pull request merged**